cluster-autoscaler/processors/nodegroupset/compare_nodegroups.go

```diff
@@ -31,6 +31,9 @@ const (
 	// MaxFreeDifferenceRatio describes how free resources (allocatable - daemon and system pods)
 	// can differ between groups in the same NodeGroupSet
 	MaxFreeDifferenceRatio = 0.05
+	// MaxMemoryDifferenceInKiloBytes describes how much memory
+	// capacity can differ but still be considered equal.
+	MaxMemoryDifferenceInKiloBytes = 2 << 7 * 1000 // 128Ki
```

Comment:
Please write it as 128000

Reply:
https://github.com/kubernetes/autoscaler/compare/a743c4a9743bcacc886d8bb61773472fb9e7f7d0..56d99b72b3384b20b33437571e605eb644cf5172